ambari-infra/ambari-infra-assembly/pom.xml

```diff
@@ -196,6 +196,10 @@
 
                     <copy file="${infra.solr.plugin.dir}/target/ambari-infra-solr-plugin-${project.version}.jar"
                           toDir="${project.build.directory}/solr/server/solr-webapp/webapp/WEB-INF/lib/"/>
+                    <copy file="${infra.solr.plugin.dir}/target/libs/ambari-metrics-common-${project.version}.jar"
+                          toDir="${project.build.directory}/solr/server/solr-webapp/webapp/WEB-INF/lib/"/>
+                    <copy file="${infra.solr.plugin.dir}/target/libs/commons-logging-1.1.1.jar"
```

Comment:
that change can be related here: https://issues.apache.org/jira/browse/AMBARI-23112
cc @avijayanhwx